stressor/README.md

```diff
@@ -9,16 +9,42 @@ It works sequentially through the list of test plans, completing the runs for on
 1. Install dependencies with `npm i`
 2. set an environment variable `GITHUB_TOKEN` with an access token. To generate a new token:
 
-* https://github.com/settings/personal-access-tokens/new
-* Generate a new token, make sure it has access to the repo you'll be running the stress actions on, and give it Read & Write "Actions" permissions (everything else can stay default).
-* For more information see [these docs](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event) for which token needs which auth scopes and how to generate them.
+- https://github.com/settings/personal-access-tokens/new
+- Generate a new token, make sure it has access to the repo you'll be running the stress actions on, and give it Read & Write "Actions" permissions (everything else can stay default).
+- For more information see [these docs](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event) for which token needs which auth scopes and how to generate them.
 
 ## Running
 
-1. It is prefered for you to run the stress test against your own personal "non-fork" of this repo (create a personal repo and push to it instead of using "fork" so it isn't part of the "network") to limit the number of action runs against the main branch.
+1. It is prefered for you to run the stress test against your own personal "non-fork" of this repo (create a personal repo and push to it instead of using "fork" so it isn't part of the "network") to limit the number of action runs against the main branch. You generally don't want to run these tests in the copy of this repo that lives in the bocoup GitHub organization.
```

Comment:
@jugglinmike Given what we're learning about the generous GH Action limits on public repos, do we still want to recommend this? Keeping a personal copy of the repo as a "non-fork" is pretty hard to maintain in the long run...